<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>39</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Codigo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCodigo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>31</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>168</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descripcion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>8</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre Especie</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNombreEspecie">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>108</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Region</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRegion">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCrear">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineReservaNatural">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>541</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTipo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>58</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Reserva Natural</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>108</y>
      <width>105</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo (Flora o Fauna)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkPeligro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>La Especie esta en Peligro de Extincion</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>1481</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMostrar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textDescripcionEspecie">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>541</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBuscar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingrese codigo para la busqueda</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nota: Primero presionar le Boton &quot;Mostrar&quot; para determinar los datos ya existentes..</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1512</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
